api/applications/applications.proto

```diff
@@ -20,6 +20,16 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
 };
 
 service Applications {
+    /**
+    * Authenticate exchanges a code received from an OAuth2 callback for a user token
```

Comment:
Let re-work this comment. It doesn't really do this in our case. This will end up as auto-generated API documentation.